code/modules/industrial_lift/tram/tram_machinery.dm

```diff
@@ -646,6 +648,20 @@ GLOBAL_LIST_EMPTY(tram_doors)
 
 	use_power(active_power_usage)
 
+	if(malfunctioning)
+		var/list/malf_sign = list("[base_icon_state][DESTINATION_WEST_ACTIVE]",
```

Comment:
Can we make this static? Doesn't make sense to rebuild the list every time we run the check.

Reply:
Yeah, sure.